google/cloud/aiplatform/training_jobs.py

```diff
@@ -66,6 +66,290 @@
 )
 
 
+class TrainingJob(base.AiPlatformResourceNoun):
+    client_class = pipeline_service_client.PipelineServiceClient
+    _is_client_prediction_client = False
+
+    def __init__(
+        self,
+        display_name: str,
+        container_uri: str,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ):
+        """Constructs a Training Job.
+
+        Args:
+            display_name (str):
+                Required. The user-defined name of this TrainingPipeline.
+            container_uri (str):
```

Comment:
Is this only relevant to Custom Training?

@sasha-gitg

Reply:
Yes.

Reply:
Ok removed from base class

Reply:
Please remove the docstring.